Xamarin.PropertyEditing.Mac/Xamarin.PropertyEditing.Mac.csproj

```diff
@@ -170,162 +171,162 @@
+    <None Include="PropertyEditingResource\Contents\Resources\pe-resource-editor-32~sel.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-resource-editor-32~sel%402x.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-sort-alphabetically-16.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-sort-alphabetically-16%402x.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-sort-alphabetically-16~dark.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-sort-alphabetically-16~dark%402x.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-down-focus-blue.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-down-focus-blue%402x.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-down-focus-blue~dark.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-down-focus-blue~dark%402x.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-down-focus-graphite.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-down-focus-graphite%402x.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-down-focus-graphite~dark.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-down-focus-graphite~dark%402x.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-down.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-down%402x.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-down~dark.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-down~dark%402x.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-up-focus-blue.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-up-focus-blue%402x.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-up-focus-blue~dark.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-up-focus-blue~dark%402x.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-up-focus-graphite.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-up-focus-graphite%402x.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-up-focus-graphite~dark.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-up-focus-graphite~dark%402x.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-up.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-up%402x.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-up~dark.png" />
+    <None Include="PropertyEditingResource\Contents\Resources\pe-stepper-up~dark%402x.png" />

```

Comment:
I left in it so that from within the IDE you can see what resources you have, otherwise you have to keep flicking to the Resource Dir to see what is there. Also makes it easy to keep things in sync with git from within the IDE. ie, if you add a resource via the IDE, it will automagically add it to the git repo.